tests/dogtag/pytest-ansible/provision/post_provision.yml

```diff
@@ -0,0 +1,21 @@
+- hosts: master
+  gather_facts: true
+
+  tasks:
+
+    - name: Install list of packages for CS Master for Fedora
+      dnf : pkg={{item}} state=latest
+      with_items:
+        - setools
+        - dnf-plugins-core
+      when: ansible_distribution == "Fedora"
+
+# The installation is breaking with pki/master with ssl handshake error.
+# Team is working on the fix. Would uncomment below as soon as it gets fixed.
+#    - name: set PKI master copr repo
```

Comment:
Critical: This has been fixed.

Note that minimum required version for JSS and tomcatjss have been bumped recently. Official builds haven't been made yet and are available only in the COPR repo. So, it needs to be enabled to pull these new packages.

Reply:
Fixed